src/StrList.cc

```diff
@@ -131,3 +131,17 @@ strListGetItem(const String * str, char del, const char **item, int *ilen, const
     return len > 0;
 }
 
+SBuf
+getListMember(const String &list, const char *key, const char delimiter)
```

Comment:
This code was in HttpHeader because it is tied directly to HTTP parsing semantics for MiME header fields. There are parsing tolerance criteria which we may have to add to this logic to enforce certain behaviours. So please do not move it here.

The requested change to the HttpHeader:: members should remove the need for this code to be touched at all by this PR.

Reply:
I am asking because this header is on the RoadMap for removal and all the code in it is going to be the focus for removal, some in ongoing branches, some later. If you can avoid adding things here it will help reduce that workload. This is not a blocker though, just a request.

Reply:
resolving this issue since it is wishlist.